File: 30 days of code/Day 02 - Operators/Operators .cs
/**
 *              OPERATORS | C#
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 02
 */

using System;
using System.Globalization;
using System.IO;

class Program
{
    static void Main(string[] args)
    {
        var reader = new InputReader(false);
        double mealCost = reader.NextDouble;
        int tipPercent = reader.NextInt;
        int taxPercent = reader.NextInt;
        reader.Close();

        double tip = (mealCost * tipPercent) / 100;
        double tax = (mealCost * taxPercent) / 100;
        int totalCost = (int)Math.Round(mealCost + tip + tax);

        Console.WriteLine(totalCost);
    }
}

class InputReader
{
    private TextReader input;

    public InputReader(bool fromFile)
    {
        input = fromFile ? new StreamReader("input.txt") : Console.In;
    }

    public string NextLine => input.ReadLine();

    public int NextInt => int.Parse(NextLine);

    public double NextDouble => double.Parse(NextLine, CultureInfo.InvariantCulture);

    public void Close() => input.Close();
}

File: 30 days of code/Day 02 - Operators/Operators .java
/**
 *              OPERATORS | JAVA
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 02
 */

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws FileNotFoundException {
        File file = new File("input.txt");
        Scanner scan = new Scanner(file);
        {
            double mealCost = Double.parseDouble(scan.nextLine());
            int tipPercent = scan.nextInt();
            int taxPercent = scan.nextInt();

            double tip = (mealCost * tipPercent) / 100;
            double tax = (mealCost * taxPercent) / 100;
            int totalCost = (int) Math.round(mealCost + tip + tax);

            System.out.println("The total meal cost is " + totalCost + " dollars.");
        }
        scan.close();
    }
}
